content/zh/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,217 @@
+resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+to manage applications and their components. Operators follow
+Kubernetes principles, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+-->
+
+
+Operator 是 Kubernetes 的扩展软件，它利用 [自定义资源](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) 管理应用及其组件。
+Operator 遵循 Kubernetes 的理念，特别是在 [control loop](/docs/concepts/#kubernetes-control-plane) 方面。
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+<!--
+## Motivation
+
+The Operator pattern aims to capture the key aim of a human operator who
+is managing a service or set of services. Human operators who look after
+specific applications and services have deep knowledge of how the system
+ought to behave, how to deploy it, and how to react if there are problems.
+
+People who run workloads on Kubernetes often like to use automation to take
+care of repeatable tasks. The Operator pattern captures how you can write
+code to automate a task beyond what Kubernetes itself provides.
+-->
+## 初衷
+
+Operator 模式旨在捕获（正在管理一个或一组服务的）运维人员的关键目标。
+负责特定应用程序和服务的运维人员，在系统应该如何运行、如何部署以及出现问题时如何处理等方面有深入的了解。
+
+在 Kubernetes 上运行工作负载的人们都喜欢通过自动化来处理重复的任务。Operator 模式捕获你是如何编写代码以实现自动化（Kubernetes 本身提供功能以外的）任务。
+
+<!--
+## Operators in Kubernetes
+
+Kubernetes is designed for automation. Out of the box, you get lots of
+built-in automation from the core of Kubernetes. You can use Kubernetes
+to automate deploying and running workloads, *and* you can automate how
+Kubernetes does that.
+
+Kubernetes' {{< glossary_tooltip text="controllers" term_id="controller" >}}
+concept lets you extend the cluster's behaviour without modifying the code
+of Kubernetes itself.
+Operators are clients of the Kubernetes API that act as controllers for
+a [Custom Resource](/docs/concepts/api-extension/custom-resources/).
+-->
+## Kubernetes 上的 Operator
+
+Kubernetes 为自动化而生。开箱即用，您可以从 Kubernetes 核心中获得许多内置的自动化功能。
+您可以使用 Kubernetes 自动化部署和运行工作负载， *甚至* 可以自动化 Kubernetes 自身。
+
+Kubernetes {{< glossary_tooltip text="控制器" term_id="controller" >}} 使您无需修改 Kubernetes 自身的代码，即可以扩展集群的行为。
+Operator 是 Kubernetes API 的客户端，充当[自定义资源](/docs/concepts/api-extension/custom-resources/)的控制器。
+
+<!--
+## An example Operator {#example}
+
+Some of the things that you can use an operator to automate include:
+
+* deploying an application on demand
+* taking and restoring backups of that application's state
+* handling upgrades of the application code alongside related changes such
+  as database schemas or extra configuration settings
+* publishing a Service to applications that don't support Kubernetes APIs to
+  discover them
+* simulating failure in all or part of your cluster to test its resilience
+* choosing a leader for a distributed application without an internal
+  member election process
+-->
+## Operator 示例 {#example}
+
+使用 Operator 可以自动化的事情包括：
+
+* 按需部署应用
+* 获取/还原应用状态的备份
+* 处理应用代码的升级以及相关改动。例如，数据库 schema 或额外的配置设置
+* 发布一个服务，要求不支持 Kubernetes API 的应用也能发现它
```

Comment:
I would consider leaving Service untranslated; it refers to the Kubernetes resource named Service.

Reply:
> I would consider leaving Service untranslated; it refers to the Kubernetes resource named Service.

I got it, thank you !